src/EightBall.elm

```diff
@@ -572,34 +568,27 @@ scratch when =
   - PlayersFault - when a player scratches (or, in the future, hits the wrong ball first). The next player must place the ball in hand.
   - NextShot - waiting for a player to shoot. Use `currentPlayer` to figure out which player is shooting. Use `playerShot` after the player shoots.
   - GameOver - when the game is over, this returns the winner.
-  - Error - when something unexpected happens, like a ball pocketed twice.
 
 -}
 type WhatHappened
     = IllegalBreak (Pool AwaitingRack)
     | PlayersFault (Pool AwaitingBallInHand)
     | NextShot (Pool AwaitingNextShot)
-      -- | NextTurn (Pool AwaitingNextTurn)
     | GameOver (Pool AwaitingNewGame) { winner : Int }
-    | Error String
 
 
 {-| Set game over via this function so we don't forget to add the internal event.
 -}
-gameOver : Player -> PoolData -> WhatHappened
-gameOver winner poolData =
-    GameOver
-        (Pool
-            { poolData
-                | events =
-                    poolData.events
-                        ++ [ { event = GameOver_
-                             , when = lastEventTime poolData.events
-                             }
-                           ]
-            }
-        )
-        { winner = playerToInt winner
+endGame : Pool a -> Pool AwaitingNewGame
```

Comment:
This is needed because elm-review complained that `AwaitingNewGame` was an unused type.